pkg/karmadactl/register/register.go

```diff
@@ -151,6 +151,13 @@ func NewCmdRegister(parentCommand string) *cobra.Command {
 		Annotations: map[string]string{
 			cmdutil.TagCommandGroup: cmdutil.GroupClusterRegistration,
 		},
+
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
I prefer to move this to `Validate` function. What u think? @XiShanYongYe-Chang

Reply:
+1

Reply:
I refer to https://github.com/karmada-io/karmada/blob/ade913173ee779a96826d24f35820ffe8d719f5d/cmd/agent/app/agent.go#L66-L73 

I think the check of args parameter should be done before Complete() @jwcesign @XiShanYongYe-Chang

Reply:
It seems to be straightforward to use `Args: cobra.MaximumNArgs(1)`.

Refer to https://github.com/kubernetes/kubernetes/blob/57eb5d631ccd615cd161b6da36afc759af004b93/cmd/kubeadm/app/cmd/join.go#L208-L209

Reply:
I accept and done @lonelyCZ